File: samples/src/main/java/tk/zielony/carbonsamples/feature/RoundedCornersActivity.java
package tk.zielony.carbonsamples.feature;

import android.os.Bundle;

import carbon.widget.ImageView;
import tk.zielony.carbonsamples.ActivityAnnotation;
import tk.zielony.carbonsamples.R;
import tk.zielony.carbonsamples.ThemedActivity;
import tk.zielony.randomdata.common.DrawableImageGenerator;

@ActivityAnnotation(layout = R.layout.activity_rounded_corners, title = R.string.roundedCornersActivity_title)
public class RoundedCornersActivity extends ThemedActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

        initToolbar();

        ImageView view = findViewById(R.id.image);
        view.setImageDrawable(new DrawableImageGenerator(this).next(null));
    }
}

File: samples/src/main/java/tk/zielony/carbonsamples/feature/RoundedCornersActivity.kt
package tk.zielony.carbonsamples.feature

import android.os.Bundle
import kotlinx.android.synthetic.main.activity_rounded_corners.*
import tk.zielony.carbonsamples.SampleAnnotation
import tk.zielony.carbonsamples.R
import tk.zielony.carbonsamples.ThemedActivity
import tk.zielony.randomdata.common.DrawableImageGenerator

@SampleAnnotation(layoutId = R.layout.activity_rounded_corners, titleId = R.string.roundedCornersActivity_title, iconId = R.drawable.ic_rounded_corner_black_24dp)
class RoundedCornersActivity : ThemedActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        initToolbar()

        image.setImageDrawable(DrawableImageGenerator(this).next())
    }
}
